Externals_CAM.cfg

```
@@ -2,7 +2,7 @@
 local_path = src/chemistry/oslo_aero
 protocol = git
 repo_url = https://github.com/NorESMhub/OSLO_AERO.git
```

Please change to allow interim testing:
```suggestion
repo_url = https://github.com/DirkOlivie/OSLO_AERO.git
```

---

I will change this and push again the PR.